components/CollapsibleList.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { View, Text, FlatList, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import fonts from '../styles/fonts';
+import colours from '../styles/colours';
+
+const itemStyle = StyleSheet.create({
+  container: {
+    paddingLeft: 30,
+    paddingVertical: 10,
+    flex: 1,
+    flexDirection: 'column',
+    borderBottomWidth: 1,
+    borderColor: colours.black,
+  },
+  text: {
+    fontSize: fonts.normal,
+    color: colours.black,
+  },
+});
+
+const renderItem = ({ item }) => (
+  <View style={itemStyle.container}>
+    <Text style={itemStyle.text}>{'Latitude: '}{item.latitude.toFixed(5)}</Text>
+    <Text style={itemStyle.text}>{'Longitude: '}{item.longitude.toFixed(5)}</Text>
+  </View>
+);
+
+const CollapsibleList = ({ data, open }) => (
+  <View>
+    { open &&
```

Comment:
This is interesting. Why not just hide the list in the parent?

Reply:
True